code/modules/fallout/obj/smelling_salts.dm

```diff
@@ -118,15 +119,4 @@
 	revived_mob.revive()
 	revived_mob.emote("gasp")
 	revived_mob.Jitter(20)
-	if(time_since_death > tlimit)
-		revived_mob.adjustOrganLoss(ORGAN_SLOT_BRAIN, max(0, min(99, ((tlimit - time_since_death) / tlimit * 100))), 150)
 	log_combat(revived_mob, revived_mob, "revived", src)
-	var/list/policies = CONFIG_GET(keyed_list/policyconfig)
-	var/memory_limit = CONFIG_GET(number/defib_cmd_time_limit)
-	var/late = memory_limit && (time_since_death > memory_limit)
-	var/policy = policies[POLICYCONFIG_ON_DEFIB_INTACT]
-	if(policy)
-		to_chat(revived_mob, policy)
-	revived_mob.log_message("revived using smelling salts, [time_since_death / 10] seconds from time of death, considered [late? "late" : "memory-intact"] revival under configured policy limits.", LOG_GAME)
-	//add_logs(user, revived_mob, "revived (smelling salts)", src)
-	in_use = FALSE
```

Comment:
The item will cease to work after you revive someone with it since the in_use var is not set to FALSE.